server/src/test/scala/services/inputfeeds/AclFeedSpec.scala

```diff
@@ -162,16 +161,18 @@ class AclFeedSpec extends CrunchTestLike {
 
       val initialAcl1 = ArrivalGenerator.apiFlight(actPax = 150, schDt = "2017-01-01T00:05Z", iata = "BA0001", status = "forecast")
       val initialAcl2 = ArrivalGenerator.apiFlight(actPax = 151, schDt = "2017-01-01T00:15Z", iata = "BA0002", status = "forecast")
-      val initialAcl = Set(initialAcl1, initialAcl2)
-      val initialLive = Set(ArrivalGenerator.apiFlight(actPax = 99, schDt = "2017-01-01T00:05Z", iata = "BA0001", status = "scheduled"))
+      val initialAcl = Flights(Seq(initialAcl1, initialAcl2))
+      val initialLive = Flights(Seq(ArrivalGenerator.apiFlight(actPax = 99, schDt = "2017-01-01T00:05Z", iata = "BA0001", status = "scheduled")))
 
       val newLive = Set(
         ArrivalGenerator.apiFlight(actPax = 105, schDt = "2017-01-01T00:05Z", estDt = "2017-01-01T00:06Z", iata = "BAW0001", status = "estimated"))
 
-      val crunch = runCrunchGraph(
-        now = () => SDate(scheduledLive),
-        initialBaseArrivals = initialAcl,
-        initialLiveArrivals = initialLive)
+      val crunch = runCrunchGraph(now = () => SDate(scheduledLive))
+
+      offerAndWait(crunch.baseArrivalsInput, initialAcl)
+      offerAndWait(crunch.liveArrivalsInput, initialLive)
+
+      Thread.sleep(1000) // Let the initial arrivals work their way through the system
```

Comment:
does this need a whole second?

Reply:
I don't know. I don't like these hacky waits, but it does fail without them. I'll try reducing to 250ms